test/spec/core/jdl_application_configuration.spec.js

```diff
@@ -100,17 +100,6 @@ describe('JDLApplicationConfiguration', () => {
     });
   });
   describe('setOption', () => {
-    context('when not passing an option', () => {
-      let configuration;
-
-      before(() => {
-        configuration = new JDLApplicationConfiguration();
-      });
-
-      it('should fail', () => {
-        expect(() => configuration.setOption()).to.throw(/^An option has to be passed to set an option\.$/);
-      });
-    });
     context('when setting a new option', () => {
```

Comment:
You've missed a branch here.
Here are the cases (compare those with the `context` paths in the test):
  - The option is passed
    - It didn't exist => it's added
    - It did exit => the value is updated
  - the option isn't passed => **nothing should happen**

The last case is missing (yep, I'm very touchy about hitting the right coverage)

Reply:
Ah, I think I know; I have added this test back. I don't think we have to remove this at all. :smile: